main/templates/main/base.html

```diff
@@ -19,6 +18,10 @@
     {% endblock extra_js %}
     <script src="{% static 'main/js/htmx.min.js' %}"></script>
     <script src="{% static 'main/js/_hyperscript.min.js' %}"></script>
+    <link rel="shortcut icon"
+          type="image/png"
+          href="{% static 'main/images/Favicon.png' %}">
+    <!DOCTYPE html>
```

Comment:
Looks like this has been moved by accident.

Reply:
If we want to add the logo to the navbar as well, would we keep the "Run Control" name in place something like this 
![image](https://github.com/user-attachments/assets/ad1da72b-46b9-42d9-b168-1e39e9f87c16)
or just remove it all together.